pkg/networkutils/mocks/network_mocks.go

```diff
@@ -133,15 +145,3 @@ func (m *MockNetworkAPIs) UseExternalSNAT() bool {
 func (mr *MockNetworkAPIsMockRecorder) UseExternalSNAT() *gomock.Call {
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "UseExternalSNAT", reflect.TypeOf((*MockNetworkAPIs)(nil).UseExternalSNAT))
 }
-
```

Comment:
AFAICT, these mocks weren't changed in content, right? Just moved for some reason?

Reply:
Yeah. I checked for a makefile target or such that would install the right version of `mockgen`, but since I didn't see one I just went with the latest, and the latest version seems to generate slightly different code.